mydumper.c

```diff
@@ -2043,8 +2043,23 @@ guint64 estimate_count(MYSQL *conn, char *database, char *table, char *field, ch
 	MYSQL_FIELD * fields = mysql_fetch_fields(result);
 
 	guint i;
+	const char* field_name = "rows";
+	if (detected_server == SERVER_TYPE_TIDB) {
+		// result of `EXPLAIN SELECT` for TiDB is different than MySQL, we fetch the first row's `count` is enough.
+		// the string format of `count` is `f`, so we can treat it as guint64.
```

Comment:
guint64 -> uint64

Reply:
:open_mouth: but we use `guint64 count=0;` in L2068